send/tinia_winpak_process.pl

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use DBI;
+use Getopt::Long qw(:config no_ignore_case);
+use Data::Dumper;
+
+my $username;
+my $password;
+my $dbname;
+my $pathToServiceFile;
+my $serviceName;
+
+GetOptions ("dbname|d=s" => \$dbname, "pathToServiceFile|p=s" => \$pathToServiceFile, "serviceName|s=s" => \$serviceName);
+
+if(!defined $dbname) {
+	print "Missing DBNAME to process service.\n";
+	exit 10;
+}
+
+if(!defined $pathToServiceFile) {
+  print "Missing path to file with generated data to process service.\n";
+  exit 11;
+}
+
+if(!defined $serviceName) {
+  print "Missing info about service name to process service.\n";
+  exit 12;
+}
+
+my $filename = "$pathToServiceFile/$serviceName";
+if(! -f $filename) {
+	print "Missing service file with data.\n";
+	exit 13;
+}
+
+my $configPath = "/etc/perun/services/$serviceName/$dbname";
+open FILE, $configPath or die "Could not open config file $configPath: $!";
+while(my $line = <FILE>) {
+	if($line =~ /^username: .*/) {
+		print "1\n";
+		$username = ($line =~ m/^username: (.*)$/x)[0];
```

Comment:
If you use 'x' as regexp flag, then whitespaces in regexp are ignored, therefore matched username will include space in the beginning.